File: Java/Exercises/CompoundInterest.java
import java.util.Scanner;

public class CompoundInterest {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        
        int years, frequency;
        double principle, amount, interest, rate;

        System.out.print("Enter Principle amount: "); principle = scan.nextDouble();
        System.out.print("Enter rate of interest (%): "); rate = scan.nextDouble()/100;
        System.out.print("Enter time period (years): "); years = scan.nextInt();
        System.out.print("Enter no of times interest calculated annualy: "); frequency = scan.nextInt();

        amount = principle * Math.pow((1 + rate/frequency), (years * frequency));
        interest = amount - principle;

        System.out.printf("Interest Applied: %.2f\n", interest);
        System.out.printf("Total payable amount: %.2f\n", amount);

        scan.close();
    }
}

File: C/Self-made/compoundInterest.c
//This program takes Principle, Rate and Time as inputs and print simple interest;
#include <stdio.h>
#include <math.h>

int main(){
    //declare variable;
    float Principle, Rate, Time, CI, Amount;

    //input values;
    printf("Input - (Principle Rate Time):");
    scanf("%f %f %f", &Principle, &Rate, &Time);

    //calculate compound interest;
    Amount = Principle*pow(1+Rate/100, Time);
    CI = Amount-Principle;

    //print output;
    printf("Compound interest is %f\n", CI);
    printf("Total Payable Amount is %f\n", Amount);

    return 0;
}